src/components/Proposals/ProposalDetails/FeedbackModule.js

```diff
@@ -69,9 +69,9 @@ function FeedbackModule({ vote, mode }) {
                 <Strong>
                   {formatTokenAmount(
                     collateral.challengeAmount,
-                    collateralToken.decimals
+                    collateral.token.decimals
                   )}{' '}
-                  {collateralToken.symbol}
+                  {collateral.token.symbol}
```

Comment:
I moved the collateral token inside the `collateral` object, I think it's clearer where to find it there.